cmd/generator/main.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"os"
+)
+
+func main() {
+	argsWithProg := os.Args
+
+	if len(argsWithProg) != 2 {
+		panic("Wrong amount of arguments!")
+	}
+
+	if len(argsWithProg[1]) == 0 {
+		panic("Given regex is empty!")
+	}
+
+	regex := argsWithProg[1]
+
+	_, err := os.Stdout.WriteString((generateCode(regex) + "\n"))
```

Comment:
actually `fmt.Prinln` also returns `err`, so it is normal, not to handle err here

```suggestion
	os.Stdout.WriteString((generateCode(regex) + "\n"))
```

If linter would be unhappy, then just add `nolint`.

Reply:
Ok :) Linter is not unhappy about this